WebApiCoreSeed.Infrastructure.Tests/RestClient/ErrorDto.cs

```diff
@@ -1,9 +1,9 @@
-﻿namespace WebApiCoreSeed.UnitTests.Infrastructure.RestClient
-{
-    public class ErrorDto
-    {
-        public string ErrorDescription { get; set; }
-
-        public int ErrorCode { get; set; }
-    }
-}
+﻿namespace WebApiCoreSeed.Infrastructure.Tests.RestClient
+{
+    public class ErrorDto
```

Comment:
This class, here, looks suspicious. Do you mind taking a look what's going on? (as a separate task, of course)

Reply:
Oh yeah, that is a class to type a > 300 HTTP code body, that helps testing the RestClient. From my side, looks like an Ok approach

Reply:
Hmm not convinced yet but don't worry I have a plan to revisit the whole project and specifically all the tests.